lib/portlayer/storage/vsphere/image.go

```diff
@@ -464,26 +441,34 @@ func (v *ImageStore) GetImage(op trace.Operation, store *url.URL, ID string) (*p
 		return nil, err
 	}
 
+	diskDsURI := v.imageDiskDSPath(storeName, ID)
+
 	var s = *store
-	var parentURL *url.URL
 
-	parentID := v.parents.Get(ID)
-	if parentID != "" {
-		parentURL, _ = util.ImageURL(storeName, parentID)
+	dsk, err := v.dm.Get(op, diskDsURI)
+	if err != nil {
+		return nil, err
+	}
+
+	var parentURL *url.URL
+	if dsk.ParentDatastoreURI != nil {
+		_, vmdk := filepath.Split(dsk.ParentDatastoreURI.Path)
```

Comment:
`path.Base` instead of `filepath.Split`.  And in general, always `path` instead of `filepath`, we don't need or want any special Windows treatment that filepath provides.